### DISCUSSION TITLE
Eclipse Search for Maven Dependencies not working

### DISCUSSION THREAD

**Discussion by thaumatrope** [eclipse-m2e/m2e-core#1598] [General]
Hello,

unfortunately, I can't get Eclipse Search working when searching for Maven Dependencies. I am using the newest Eclipse Build for Java Developers.
I have added two screenshots. I think I have made all the necessary settings in Eclipse Preferences regarding Maven. But when I try to resolve nothing happens. According to others, there should be a workspace-specific download after restarting the IDE when the setting "Download repository index updates on startup" is activated. Should be a somewhat bigger file, but nothing is happening. Even in the status bar, no download is indicated. I have restarted the IDE multiple times. Could someone please assist?

![Eclipse Maven](https://github.com/eclipse-m2e/m2e-core/assets/3491153/f7ff051b-b43f-4e97-b208-75ca7a35439b)

**Comment by JeremyBloom** [comment]
I have the same problem. This must be a new issue, since I'm using Eclipse 2023-09 (4.29.0)

**Comment by 3twgrasdv** [comment]
Same problem, cant find solution on mr google

**Comment by billson-my** [comment]
having the same with Eclipse version 2023-12R.
environment together with Eclipse version:
- 2023 09
- 2023 06

**Comment by NEXT-daniel-velkov** [comment]
Same here. Tried a lot of things but to no avail.

**Comment by giulio-quaresima** [comment]
Same problem.

```
Version: 2024-03 (4.31.0)
Build id: 20240307-1437
```

**Comment by solf** [comment]
I was able to find information elsewhere: https://stackoverflow.com/questions/77339316/maven-index-not-available-in-eclipse#comment136343651_77339316

It looks like the functionality if half-broken atm -- if you want to search for something, you should do it in text view by typing something in the <dependencies> section (or <plugins>) and hitting auto-complete.

I tried it and it seems to work.

However I wasn't able to figure out how to see available versions there (except for the latest). If someone knows how, I'd like to know.

If that's impossible now, I feel it's a significant step backwards in functionality compared to the previous situation.

**Comment by Geolykt** [comment]
For reference this is completely intended behaviour, as caused by https://github.com/eclipse-m2e/m2e-core/issues/509
Why the relevant views weren't removed too is a bit beyond me, but I suppose it is what it is.

**Comment by JeremyBloom** [comment]
Sorry, but this isn't really an answer. You say there are better alternatives available, but what are they? I don't want or need to be a Maven expert; I just want to search for my dependencies without having to know the the Maven coordinates for them. You need to do a better job of explaining how to do that now that the indexer is gone. However, I've already moved on to another development environment and won't be returning to Eclipse.
